ui/react-components/config.js

```diff
@@ -19,4 +20,7 @@ export const recurringAppointmentsConflictsUrl = reactConfig["recurringAppointme
 export const i18nConfigPath = reactConfig["i18nConfigPath"];
 export const BAHMNI_CONFIG_URL = reactConfig["BAHMNI_CONFIG_URL"];
 export const IMPLEMENTATION_CONFIG_URL = reactConfig["IMPLEMENTATION_CONFIG_URL"];
-export const locationTagName = reactConfig["locationTagName"];
\ No newline at end of file
+export const locationTagName = reactConfig["locationTagName"];
+export const otpServiceUrl = reactConfig["otpServiceUrl"];
+export const clinicName = reactConfig["clinicName"];
```

Comment:
Same comment as earlier. Cant read this value from code config. Should come from external config.

Reply:
@kavitha-sundararajan please refer to helper.js file to fetch external configuration parameters

Reply:
refactored clinicName to be read from login location